.gitignore

```diff
@@ -5,6 +5,7 @@
 bin/
 build/
 ci/
+jacoco.exec
 
 # IntelliJ
```

Comment:
Do you need to add more to the gitignore?
At least 
```
# Local configuration file (sdk path, etc)
local.properties
```

I was looking at Edge and few more can be added.